assets/data/host.js

```diff
@@ -0,0 +1,76 @@
+    Quisque hendrerit lectus in quam condimentum, et molestie libero fringilla. Cras nec nisi id urna condimentum vehicula vel a justo. Phasellus quis risus sem. Fusce euismod eros eros, sit amet convallis tortor bibendum id. Quisque mi neque, viverra sed sagittis sed, mollis id lacus. Vivamus at est lorem. Nulla eget ante efficitur, lacinia ante quis, hendrerit lacus.
+`;
+
+export const shortActivityDescription = `
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentesque leo magna, blandit eget nisi a, posuere feugiat purus. In quis porttitor nulla, vitae lobortis neque. Quisque at posuere justo, quis laoreet sapien. Mauris pulvinar iaculis nisi. Donec iaculis ante accumsan, placerat elit non, congue urna.
+`;
+
+export const libraryModelList = `
+    <option>[5] Example</option>
+    <option>[5] Template</option>
+    <option>[4] Client Perspective Example</option>
+    <option selected>[4] Disease</option>
+    <option>[4] Function</option>
+    <option>[4] Investments</option>
+    <option>[4] Predator Prey Game</option>
+    <option>[4] Root Beer Game</option>
+    <option>[3] Bug Hunters Camouflage</option>
+    <option>[3] Disease With Android Avoidance</option>
+    <option>[3] Minority Game</option>
+    <option>[3] Tragedy of the Commons</option>
+    <option>[3] Walking</option>
+    <option>[2] Disease Doctors</option>
+    <option>[2] Memory</option>
+    <option>[2] Oil Cartel Alternate</option>
+    <option>[2] Public Good</option>
+    <option>[1] BeeSmart</option>
+    <option>[1] Bug Hunters Adaptations</option>
+    <option>[1] Bug Hunters Competition</option>
+    <option>[1] Dice Stalagmite</option>
+    <option>[1] Fish Spotters</option>
+    <option>[1] Gridlock</option>
+    <option>[1] Guppy Spots</option>
+    <option>[1] PANDA BEAR</option>
+    <option>[1] Polling</option>
+    <option>[1] Polling Advanced</option>
+    <option>[1] Prisoners Dilemma</option>
+
+    <option>[a] Oil Cartel</option> <!-- requires narrowcast plots -->
+
+    <option>[0] Restaurants</option> <!-- makes no distinction between AI restaurants and player-owned ones -->
+
+    <option>[0] Bird Breeders</option> <!-- requires spectator clients -->
+    <option>[0] Gridlock Alternate</option> <!-- requires spectator clients -->
+
+    <option>[0] Client Overrides Example</option> <!-- requires view overrides -->
+    <option>[0] Critter Designers</option> <!-- requires view overrides -->
+    <option>[0] Current in a Wire</option> <!-- requires view overrides -->
+    <option>[0] MTG 1 Equal Opportunities</option> <!-- requires view overrides -->
+    <option>[0] MTG 2 Random Assignment</option> <!-- requires view overrides -->
+    <option>[0] MTG 3 Feedback Loop</option> <!-- requires view overrides -->
+    <option>[0] Sampler</option> <!-- requires view overrides -->
+`;
+
+export const cclDiseaseActivityUrl = `
```

Comment:
My export style is:

  * If there's only one export from a file, you can directly do `export const whatever = 123`
  * Otherwise, it should be `const whatever1 = "apples"; const whatever2 = "oranges"; export { whatever1, whatever2 };`

Reply:
Decided not a have a separate file for exports, but I'll keep this in mind for the future - thanks.